cmd/fleet/main.go

```diff
@@ -2,7 +2,8 @@ package main
 
 import (
 	"github.com/mhausenblas/kcf/cmd/fleet/cli"
-	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp" // required for GKE
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"  // required for GKE
```

Comment:
Isn't this enough for all auth providers?
```go
_ "k8s.io/client-go/plugin/pkg/client/auth"
```

Reply:
Yeah, good point! @naemono do you want to test this?